### PR TITLE
E2E: Disable WSL-integrations test.

### PR DESCRIPTION
**Pull request by mook-as**
That page has moved to preferences.

### CODE REVIEW

File: e2e/wsl-integrations.e2e.spec.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { createDefaultSettings, packageLogs, reportAsset } from './utils/TestUti
 import { spawnFile } from '@/utils/childProcess';
 
 test.describe('WSL Integrations', () => {
+  test.skip(true, 'TODO');
```
Comment (rak-phillip) [inline e2e/wsl-integrations.e2e.spec.ts:18]: I guess this kinda serves the same purpose as a comment with a TODO in it. Do we have an issue that we reference here so we can understand what needs to change when we come back later?

**Review by rak-phillip — COMMENTED** [review]
Honestly, LGTM. Just a nitpick about `TODO` in code 🙂

File: e2e/wsl-integrations.e2e.spec.ts (hunk 1 above)
Comment (mook-as) [reply e2e/wsl-integrations.e2e.spec.ts]: Filed #2881 and added that to the comment in c36b69c5cb587d972ba5f43f240a3caca5799b54.

**Review by rak-phillip — APPROVED** [review]
LGTM - waiting for CI to pass

**Pull request merged**